.github/workflows/python-app.yml

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Lucca Baumgärtner <lucca.baumgaertner@fau.de>
+
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python application
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  license:
+    name: License check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
```

Comment:
Should the python Version be 3.10 everywhere?

Reply:
I think so too, classic copy paste mistake. Thanks for pointing it out!

Reply:
Feel free to resolve this comment once it is changed.